src/components/common/Extensions/ExtensionUpgradeButton/ExtensionUpgradeButton.tsx

```diff
@@ -42,20 +42,39 @@ const ExtensionUpgradeButton = ({ extensionData }: Props) => {
   if (!user || extensionData.isDeprecated || !extensionData.isInitialized) {
     return null;
   }
-  // @TODO check user permissions for canUpgrade - hasRoot(allUserRoles)
+
   const canUpgrade = true;
 
   return (
     <ActionButton
-      appearance={{ theme: 'primary', size: 'medium' }}
       submit={ActionTypes.EXTENSION_UPGRADE}
       error={ActionTypes.EXTENSION_UPGRADE_ERROR}
       success={ActionTypes.EXTENSION_UPGRADE_SUCCESS}
       transform={transform}
-      text={{ id: 'button.upgrade' }}
+      text={{ id: 'button.updateVersion' }}
       disabled={!isSupportedColonyVersion || !extensionCompatible || !canUpgrade}
+      onSuccess={() =>
```

Comment:
maybe will be better if you add toast component inside action button instead pass it as prop

Reply:
As I can see Action Button is used in other components as well (this is an existing component that I moved to Extensions). I think it's not a good idea to put toast inside the Action Button because in other cases, action on success and error could be different than showing toast.